containers/runascat/Dockerfile

```diff
@@ -5,8 +5,8 @@ description="R 3.3.2 image for use in CAW" \
 maintainer="maxime.garcia@scilifelab.se"
 
 # Install libraries
-RUN echo "r <- getOption('repos'); r['CRAN'] <- 'http://cran.us.r-project.org'; options(repos = r);" > ~/.Rprofile \
-&& R --no-save "install.packages('RColorBrewer')"
+RUN echo "r <- getOption('repos'); r['CRAN'] <- 'http://cran.us.r-project.org'; options(repos = r);" > ~/.Rprofile
+RUN Rscript -e "install.packages('RColorBrewer')"
```

Comment:
It makes a container smaller when combining as much RUN statements as possible into one.
That's why we add this `/ &&`